docs/source/installing_docker.rst

```diff
@@ -79,6 +79,28 @@ You need to create the database schema by executing::
         docker exec -it kiwi_web /Kiwi/manage.py createsuperuser
 
 
+.. _configure-kiwi-base-url:
```

Comment:
This is a named reference pointing to the section below. Is it used anywhere else? If not remove it.

Reply:
It is used in configuration.rst but I've renamed it.